### DISCUSSION TITLE
CamillaDSP

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#874] [General]
[CamillaDSP](https://henquist.github.io/0.6.3/) is now available with latest image files.

![camilladsp](https://user-images.githubusercontent.com/10094236/162214482-8a90b8ce-6702-4ea4-8eed-6883b39bfd15.jpg)

**Comment by Slaughthammer** [comment]
I'm just trying to play around with this to see if this could replace my ecasound setup, but I don't get it... can anyone point me to some documentation?
